### PR TITLE
Remove plug comp crd init. Move to latest commit in go-sdk.

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Wrt https://github.com/dapr/dapr/pull/5299

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Review by pravinpushkar — APPROVED** [review]
lgtm

**Review by shubham1172 — APPROVED** [review]

**Review by pravinpushkar — APPROVED** [review]
lgtm

File: pkg/kubernetes/upgrade.go [hunk 1]
```diff
@@ -35,15 +35,13 @@ var crds = []string{
 	"configuration",
 	"subscription",
 	"resiliency",
-	"pluggable-components",
```
Comment (DeepanshuA) [inline pkg/kubernetes/upgrade.go:38]: I may be missing the background, but this is removed intentionally?

Comment (DeepanshuA) [reply pkg/kubernetes/upgrade.go:38]: Ok, got it, just read discord's chat.

**Comment by DeepanshuA** [comment]
LGTM.

**Review by DeepanshuA — APPROVED** [review]

File: pkg/kubernetes/upgrade.go (hunk 1 above)
Comment (shubham1172) [reply pkg/kubernetes/upgrade.go:38]: It's also here FYI https://github.com/dapr/dapr/issues/5210#issuecomment-1258479834

**Pull request merged**